packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -409,28 +409,206 @@ StructuredListMediaCalendar.argTypes = {
     control: 'object',
   },
 }
-StructuredListMediaCalendar.args = { //todo: update text in samples to match calendar text
+StructuredListMediaCalendar.args = {
   showHeader: false,
   listItems: [
     {
-      content: "<cbp-grid grid-template-columns='1fr 5fr' gap='1rem'><div class='cbp-structured-list-calendar-item'><span class='cbp-structured-list-calendar-item-month'>apr</span><span class='cbp-structured-list-calendar-item-day'>31</span></div><div><cbp-typography tag='p'><cbp-icon name='arrow-right' color='var(--cbp-link-color)' size='1rem'></cbp-icon><cbp-link href='#' target='_self' style='margin-inline-start: 0.5rem'>Find the Hidden Beetle Training</cbp-link></cbp-typography><cbp-typography tag='p'>This training session will help you find those pesky hiding beetles. Inside boxes, on top of containers, or in someone's pocket, these bugs don't stand a chance of getting to America's sweet, delicious forests</cbp-typography><cbp-typography tag='p'><cbp-icon name='globe' size='1rem'></cbp-icon> <i>https://www.cbp.dhs.gov/news/beetles</i></cbp-typography></div><cbp-grid>",
+      content: `<cbp-grid grid-template-columns='1fr 5fr' gap='1rem'>
+                  <cbp-flex
+                  sx='{
+                        "width":"6.25rem",
+                        "height":"6.25rem",
+                        "display":"flex",
+                        "flex-direction":"column",
+                        "align-items":"center"
+                      }'
+                >
+                  <cbp-flex-item
+                  sx='{
+                        "width":"100%",
+                        "text-align":"center",
+                        "border-top-left-radius":"var(--cbp-border-radius-soft)",
+                        "border-top-right-radius":"var(--cbp-border-radius-soft)",
+                        "padding-block":"var(--cbp-space-2x)",
+                        "background-color":"var(--cbp-color-red-50)",
+                        "border":"1px solid var(--cbp-color-red-50)",
```

Comment:
border size 1px should have a token value.

Reply:
updated to use var(--cbp-border-size-sm)